WebApi/WebApi/Controllers/CommentController.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using WebApi.Models.DTO;
+using WebApi.Services.Services_Interfaces;
+
+
+namespace WebApi.Controllers
+{
+    [ApiController]
+    [Route("api/comment")]
+    public class CommentController : ControllerBase
+    {
+        private readonly ICommentService _commentService;
+        private readonly ILogger<CommentController> _logger;
+        public CommentController(ILogger<CommentController> logger, ICommentService commentService)
+        {
+            _commentService = commentService;
+            _logger = logger;
+        }
+        [HttpGet]
+        public ActionResult<IQueryable<CommentDTO>> GetAll([FromHeader] int userId)
+        {
+            var result = _commentService.GetAll(userId);
+            if (result == null) return NotFound();
+            return Ok(result);
+        }
+        [HttpGet("{id}")]
+        public ActionResult<CommentDTO> GetById([FromRoute] int id, [FromHeader] int userId)
+        {
+            var result = _commentService.GetById(id, userId);
+            if (result == null) return NotFound();
+            return Ok(result);
+        }
+
+        [HttpPost]
+        public async Task<ActionResult<int>> AddComment([FromBody] CommentDTO comment, [FromHeader] int userId)
+        {
+            var result = await _commentService.AddCommentAsync(userId, comment);
+            if (result == null) return NotFound();
+            return Ok(result.CommentID);
+
+        }
+        [HttpDelete("{id}")]
+        public ActionResult<bool> DeleteComment([FromRoute] int id, [FromHeader] int userId)
+        {
+            var result = _commentService.DeleteComment(id, userId);
+            if (!result) return NotFound();
+            return Ok();
+
+        }
+        [HttpPut("{id}")]
+        public async Task<ActionResult<CommentDTO>> EditComment([FromRoute] int id, [FromHeader] int userId, [FromBody] CommentDTO comment)
```

Comment:
dlaczego zwracane jest CommentDTO ? 
niezgodność ze specyfikacją: jeśli zrobisz samo OK() to on sobie dopasuje do zwracanego typu
tutaj ma być Task<ActionResult<bool>>